<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmpouleVue</class>
 <widget class="QMainWindow" name="AmpouleVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AmpouleVue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lEtat">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images/images.qrc">:/td6/ampouleEteinte2.PNG</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
      <item>
       <widget class="QPushButton" name="bAllumer">
        <property name="text">
         <string>Allumer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bEteindre">
        <property name="text">
         <string>Ã‰teindre</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>393</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
